run_tests_with_dekorate.sh

```diff
@@ -1,4 +1,16 @@
 #!/usr/bin/env bash
 
+SB_VERSION_SWITCH=""
```

Comment:
I don't like to update this script only for this example. Instead, I would change all the scripts for all the examples with a more general purpose change: this is allowing to provide any `-Dkey=value` arguments.

Reply:
Updated the script so it passes all the parameters. Also updated the docs with info on how to override the parameters.